libcloud/httplib_ssl.py

```diff
@@ -163,9 +163,10 @@ class LibcloudConnection(LibcloudBaseConnection):
     response = None
 
     def __init__(self, host, port, **kwargs):
-        self.host = '{0}://{1}'.format(
+        self.host = '{0}://{1}{2}'.format(
             'https' if port == 443 else 'http',
```

Comment:
Do we have access to `secure` argument here?

Because if we do, it looks like we still don't correctly handle one scenario - https on a non default port. For example, secure=True, port=8080 should result in `https://<host>:8080`.

Or am I missing something?